components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/gateway/GatewayAPIDTO.java

```diff
@@ -20,6 +20,10 @@
 
 package org.wso2.carbon.apimgt.api.gateway;
 
+import org.wso2.carbon.apimgt.api.model.VHost;
+
+import org.wso2.carbon.apimgt.api.model.VHost;
```

Comment:
Shall we remove the duplicate entry here?

Reply:
Done with https://github.com/wso2/carbon-apimgt/pull/13284/commits/0bc6811f75be9d19250e3e2db0736192f12caf46